### ISSUE TITLE
I can't type Russian letters

### ISSUE THREAD

**Issue by kalembet** [Swordfish90/cool-retro-term#537]
Dear Sir!

When I try to type Russian, CRT prints English letters, while MacOS terminal is OK with Russian. Can I fix it?

Thanks

**Comment by gavr123456789** [comment]
i can 
![image](https://user-images.githubusercontent.com/30507409/62674724-60240e80-b9ac-11e9-9b46-438146bf8338.png)

**Comment by jphebus** [comment]
I can, as well. Maybe try a different font? I tried typing in HD: Terminus (Modern) and System: DejaVu Sans Mono and both fonts rendered Russian text correctly.

**Comment by kalembet** [comment]
Still doesn't work. I've tried all the fonts. Still can't type in Russian :(

**Comment by gavr123456789** [comment]
@kalembet Just installed it to check special for u again. Arch linux.

**Comment by kalembet** [comment]
Yes sir. I see that it works but don't understand why it doesn't in my case? I use Mac OS Catalina. If I run nano in Terminal App everything is fine but in CRT nano shows Cyrillic symbols but doesn't type.
